testutil/integration/app.go

```diff
@@ -526,12 +526,17 @@ func NewCompleteIntegrationApp(t *testing.T, opts ...IntegrationAppOptionFn) *Ap
 		authority.String(),
 		accountKeeper,
 		bankKeeper,
+		gatewayKeeper,
```

Comment:
The `gatewayKeeper` seems to be useless

Reply:
As discussed, this will be used to support `MsgClaimMorseGateway` in a future PR.

Reply:
#PUC with a TODO_MAINNET / TODO_UPNEXT to communicate that to the reviewer + interim code readers.